full-service/src/util/b58/errors.rs

```diff
@@ -1,29 +1,29 @@
-use displaydoc::Display;
+use thiserror::Error;
```

Comment:
Whats the reason for this change?

Reply:
FYI the reason we did not use `thiserror` in the `mobilecoin` repo is that it is not no_std compatible and cannot be used in enclaves or any other no_std code (e.g. wasm).
This might not affect `full-service` but most/all error types in `mobilecoin` do not implement `std::error::Error`, and if there are ever any parts in this repository that will need to be no_std, `thiserror` will have to be removed from them.

Reply:
aaah, I see. I don't expect Full Service to ever run into that requirement, but if you think it's worth avoiding it altogether, I am fine with that.

Reply:
will revert this regardless and address thiserror stuff at a later point